### DISCUSSION TITLE
Is there any way to preserve the original nested structure when viewing through "Folders" ?

### DISCUSSION THREAD

**Discussion by chogovadze** [photoprism/photoprism#1878] [Q&A] [answered]
Hello everyone,

I am wondering if there is any way to preserve the original structure of my folders when viewing through the "Folder" tab ? This is already the case when viewing through "Library" -> "Originals". However, viewing through the library is cumbersome due to the fact that the default behaviour opens the metadata and not the file itself.

I hope this example would make my point clearer. Let's assume the structure is the following:

```
2021/
├─ Project A/
│  ├─ Videos/
│  ├─ Photos/
2022
├─ Project B/
│  ├─ Videos/
│  ├─ Photos/
```
Currently, when I look into the "Folder"'s tab I directly see thumbnails with "Videos", "Videos", "Photos", "Photos" with the path written beneath. However, I would actually like to be able to see "2021" and "2022" and go down the tree similarly to what "Library/Originals" show. Is there anything currently that would allow me to achieve this behaviour ?

Thank you for your time!

**Comment by graciousgrey** [comment, marked as answer]
That's planned: https://github.com/photoprism/photoprism/issues/1536

**Reply by chogovadze** [reply]
Thank you for the pointer and sorry for the duplicate discussion!

**Reply by graciousgrey** [reply]
You're welcome :)
